test/e2e/storage/drivers/in_tree.go

```diff
@@ -1185,7 +1184,6 @@ type vSphereDriver struct {
 
 type vSphereVolume struct {
 	volumePath string
-	nodeInfo   *vspheretest.NodeInfo
 }
 
 var _ storageframework.TestDriver = &vSphereDriver{}
```

Comment:
@dims 

I think it's important to **remove** implementation of these interfaces:

* `PreprovisionedVolumeTestDriver`
* `InlineVolumeTestDriver`
* `PreprovisionedPVTestDriver`

Otherwise the test framework will see they're implemented and will try to run these tests, which can't work without the cloud provider.

Only `DynamicPVTestDriver` interface should be implemented, because in-tree vSphere StorageClass will be translated to CSI and it can be tested without the cloud SDK linked anywhere.

Reply:
Sounds good @jsafrane is there someone who can help with this? if not please let me now

Reply:
I created https://github.com/kubernetes/kubernetes/pull/123036 to prune the interfaces.